packages/lesswrong/components/votes/VoteAgreementIcon.tsx

```diff
@@ -64,23 +64,24 @@ const styles = (theme: ThemeType) => ({
     opacity: 1,
     fontSize: '80%',
     position: 'absolute',
-    left: 4,
+    left: -2,
     top: 0,
+    pointerEvents: 'none'
   },
   smallArrowBigVoted: {
     opacity: 0.6,
     position: 'absolute',
     height: 14,
-    scale: 1.4,
-    transform: 'translate(-2.5px, 2.3px)'
+    transform: 'scale(1.4) translate(-4.1px, 2.3px)',
+    pointerEvents: 'none',
```

Comment:
Should I be scared of this?

Reply:
Yes. All the CSS positioning is a little scary.